includes/Core/Authentication/User_Input_State.php

```diff
@@ -1,78 +0,0 @@
-<?php
-/**
- * Class Google\Site_Kit\Core\Authentication\User_Input_State
- *
- * @package   Google\Site_Kit
- * @copyright 2021 Google LLC
- * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
- * @link      https://sitekit.withgoogle.com
- */
-
-namespace Google\Site_Kit\Core\Authentication;
-
-use Google\Site_Kit\Core\Storage\User_Setting;
-
-/**
- * Class representing user_input_state for the user.
- *
- * @since 1.20.0
- * @access private
- * @ignore
- */
-final class User_Input_State extends User_Setting {
-
-	/**
-	 * User option key.
-	 */
-	const OPTION = 'googlesitekit_user_input_state';
-
-	/**
-	 * Value required key.
-	 */
-	const VALUE_REQUIRED = 'required';
-
-	/**
-	 * Value completed key.
-	 */
-	const VALUE_COMPLETED = 'completed';
-
-	/**
-	 * Value missing key.
-	 */
-	const VALUE_MISSING = 'missing';
-
-	/**
-	 * Registers the setting in WordPress.
-	 *
-	 * @since 1.90.0
-	 */
-	public function register() {
-		parent::register();
-
-		add_action(
-			'googlesitekit_user_input_set',
```

Comment:
This action should also be removed in `User_Input::set_answers` as it was only intended as [a temporary workaround](https://github.com/google/site-kit-wp/pull/6155#discussion_r1042860381) and no longer necessary going forward.